homeassistant/components/poolsense/sensor.py

```diff
@@ -0,0 +1,112 @@
+"""Sensor platform for the PoolSense sensor."""
+import logging
+
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+from . import get_coordinator
+from .const import ATTRIBUTION
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS = {
+    "Chlorine": {"unit": "mV", "icon": "mdi:pool", "name": "Chlorine"},
+    "pH": {"unit": "", "icon": "mdi:pool", "name": "pH"},
+    "Battery": {"unit": "%", "icon": "mdi:battery", "name": "Battery"},
+    "Water Temp": {
+        "unit": "°C",
+        "icon": "mdi:coolant-temperature",
+        "name": "Temperature",
+    },
+    "Last Seen": {"unit": "", "icon": "mdi:clock", "name": "Last Seen"},
+    "Chlorine High": {"unit": "mV", "icon": "mdi:pool", "name": "Chlorine High"},
+    "Chlorine Low": {"unit": "mV", "icon": "mdi:pool", "name": "Chlorine Low"},
+    "pH High": {"unit": "", "icon": "mdi:pool", "name": "pH High"},
+    "pH Low": {"unit": "", "icon": "mdi:pool", "name": "pH Low"},
```

Comment:
if unit is not set, None is likely preferable to ""

Reply:
Noted and corrected.